consensus/state_sync/server.go

```diff
@@ -72,8 +69,10 @@ func (m *stateSync) HandleGetBlockRequest(blockReq *typesCons.GetBlockRequest) e
 	}
 
 	// get block from the persistence module
-	block, err := m.getBlockAtHeight(blockReq.Height)
+	blockStore := m.GetBus().GetPersistenceModule().GetBlockStore()
+	block, err := blockStore.GetBlock(blockReq.Height)
 	if err != nil {
+		m.logger.Error().Err(typesCons.ErrConsensusMempoolFull).Msg(typesCons.DisregardHotstuffMessage)
```

Comment:
Why are we logging a `ErrConsensusMempoolFull` error here?

Reply:
Good catch, this was an artifact of the refactor that I was apparently context-blind to. Updated to return a better error. 👍